subsystem/kubeapi_test.go

```diff
@@ -1112,8 +1112,12 @@ var _ = Describe("[kube-api]cluster installation", func() {
 		Eventually(func() bool {
 			return getClusterDeploymentCRD(ctx, kubeClient, clusterKey).Spec.Installed
 		}, "1m", "2s").Should(BeTrue())
-		Expect(getClusterDeploymentCRD(ctx, kubeClient, clusterKey).Status.APIURL).Should(Equal("https://api.test-cluster-sno.hive.example.com:6443"))
-		Expect(getClusterDeploymentCRD(ctx, kubeClient, clusterKey).Status.WebConsoleURL).Should(Equal("https://console-openshift-console.apps.test-cluster-sno.hive.example.com"))
+		Eventually(func() string {
```

Comment:
This is done to prevent test failures for non-related reasons, see https://issues.redhat.com/browse/MGMT-6023

Will probably open a separate PR and remove it from this one, in case it gets merged first

Reply:
https://github.com/openshift/assisted-service/pull/1652